monai/utils/module.py

```diff
@@ -27,9 +27,12 @@
 from re import match
 from types import FunctionType, ModuleType
 from typing import Any, Iterable, cast
-
 import torch
 
+import importlib.metadata
+from packaging import version
```

Comment:
please remove the import here since there is already `optional_import` below.

Reply:
Hello there,
Removing line 32 "import importlib.metadata" works fine.
However, removing line 33 "from packaging import version" causes issues since there are references for 'version' in the script...
Please advise

Reply:
I think the point here is that `version` gets imported elsewhere within the body of functions using `optional_import` so it doesn't need importation here. If it does need to be here for type checking reasons, you may need to follow a pattern [like this ](https://github.com/Project-MONAI/MONAI/blob/f1ef3e88f4bc594779c0c4188883f78bf6d1efff/monai/handlers/logfile_handler.py#L22) where the `TYPE_CHECKING` variable is used to guard the import statement.

Reply:
Another issue is that "version" is a bit problematic as a name since a number of argument variables in functions below are "version".